Common/Base/OpenGL/BaseManager.cpp

```diff
@@ -3,107 +3,53 @@
 	@author		Albert Semenov
 	@date		05/2009
 */
-
-
 #include "Precompiled.h"
-#include <windows.h>
+#include "BaseManager.h"
+#include "MyGUI_Diagnostic.h"
+
+#include <SDL_image.h>
 
 #ifdef MYGUI_CHECK_MEMORY_LEAKS
 #	undef new
 #	undef delete
 #endif
 
-#include <gdiplus.h>
-#include "BaseManager.h"
-
-#include <GL/gl.h>
-
-//for image loader
-#include <malloc.h>
-
-#ifdef __MINGW32__
-using namespace Gdiplus;
-#endif
+////for image loader
+//#include <malloc.h>
```

Comment:
If the include isn't in use anymore, then it should be removed completely, instead of being commented out.